INSTALL.md

```diff
@@ -1,13 +1,13 @@
 # Install
 
-- [Unix like](#unix)
+- [GNU/Linux (Unix-like)](#unix)
```

Comment:
Why do you make this change? [Unix-like](https://en.wikipedia.org/wiki/Unix-like) includes GNU/Linux.